### ISSUE TITLE
npm ERR! Failed resolving git HEAD

### ISSUE THREAD

**Issue by monolithed** [polyfillpolyfill/polyfill-service#331]
```
➜ npm install polyfill-service

> fsevents@0.3.5 install 
node_modules/grunt-browserify/node_modules/watchify/node_modules/chokidar/node_modules/fsevents
> node-gyp rebuild

child_process: customFds option is deprecated, use stdio instead.
  SOLINK_MODULE(target) Release/.node
  SOLINK_MODULE(target) Release/.node: Finished
  CXX(target) Release/obj.target/fse/fsevents.o
  SOLINK_MODULE(target) Release/fse.node
  SOLINK_MODULE(target) Release/fse.node: Finished
npm WARN deprecated node.extend@1.0.0: 0.0.1
|
> node-sass@2.0.1 install 
node_modules/grunt-sass/node_modules/node-sass
> node scripts/install.js


> node-sass@2.0.1 postinstall 
node_modules/grunt-sass/node_modules/node-sass
> node scripts/build.js

`darwin-x64-node-0.12` exists; testing
Binary is fine; exiting
npm WARN engine hawk@0.10.2: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine sntp@0.1.4: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine cryptiles@0.1.3: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine boom@0.3.8: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine hoek@0.7.6: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine hawk@0.10.2: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine hawk@0.10.2: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine sntp@0.1.4: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine cryptiles@0.1.3: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine hoek@0.7.6: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine sntp@0.1.4: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine cryptiles@0.1.3: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine boom@0.3.8: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine boom@0.3.8: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})
npm WARN engine hoek@0.7.6: wanted: {"node":"0.8.x"} (current: {"node":"0.12.0","npm":"2.5.1"})

> fsevents@0.3.1 install 
node_modules/grunt-polyfill-service/node_modules/polyfill-service/node_modules/6to5/node_modules/chokidar/node_modules/fsevents
> node-gyp rebuild

child_process: customFds option is deprecated, use stdio instead.
  SOLINK_MODULE(target) Release/.node
  SOLINK_MODULE(target) Release/.node: Finished
  CXX(target) Release/obj.target/fse/fsevents.o
  SOLINK_MODULE(target) Release/fse.node
  SOLINK_MODULE(target) Release/fse.node: Finished

npm ERR! Failed resolving git HEAD (https://github.com/matthew-andrews/fetch) fatal: bad object 9fe698067f2667f0dcc9aca6e56fcd14274143cf
npm ERR! Failed resolving git HEAD (https://github.com/matthew-andrews/fetch) 
npm ERR! Darwin 14.1.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code 128

npm ERR! Command failed: git rev-list -n1 9fe698067f2667f0dcc9aca6e56fcd14274143cf
npm ERR! fatal: bad object 9fe698067f2667f0dcc9aca6e56fcd14274143cf
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     npm-debug.log
```

What is it?

```
npm ERR! Failed resolving git HEAD (https://github.com/matthew-andrews/fetch) fatal: bad object 9fe698067f2667f0dcc9aca6e56fcd14274143cf
```

**Comment by triblondon** [comment]
This is a dependency of haikro, which I removed from the dep list some time ago.  I'm still planning to do a release to npm and the CDN this week, if I can only find the time.

**Comment by gobwas** [comment]
Please, I need it work too!

**Comment by monolithed** [comment]
@triblondon,  I can not update my local npm repository for this one. Really, it blocks my work.

**Comment by matthew-andrews** [comment]
perhaps i can fix it in the dependency

**Comment by triblondon** [comment]
Sorry folks, I've got a ton of test failures now, no deploy till we fix these:

https://travis-ci.org/Financial-Times/polyfill-service/builds/53358352#L821

**Comment by triblondon** [comment]
Now released as 1.3.0-rc.1

**Comment by gobwas** [comment]
Thats a good news!
